examples/multimodal/speech_llm/conf/modular_audio_gpt_config_cross_llama_lhotse.yaml

```diff
@@ -0,0 +1,361 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
could you put this file under `conf/bestow/*` ?

Reply:
Done